Classes/WPEditorViewController.m

```diff
@@ -1147,18 +1160,27 @@ - (void)alignFull
 - (void)setBold
 {
     [self.editorView setBold];
+    if (!self.editorView.isInVisualMode) {
```

Comment:
Just to reduce code redundancy a bit, and it's a small gain: but what about creating a method that contains line 1163 - 1165, so we can just call that method in all repetitions below?